tests/llmobs/suitespec.yml

```diff
@@ -92,3 +95,16 @@ suites:
       - tests/snapshots/tests.contrib.openai.*
     runner: riot
     snapshot: true
+  langgraph:
+    parallelism: 3
+    paths:
+      - '@bootstrap'
+      - '@core'
+      - '@tracing'
+      - '@contrib'
+      - '@langgraph'
+      - '@requests'
+      - '@llmobs'
+      - tests/contrib/langgraph/*
+    runner: riot
+    snapshot: false
```

Comment:
Let's set this to true, I believe the patch test case for get_version() requires a testagent